Formula/telnet.rb

```diff
@@ -39,7 +38,7 @@ def install
                    "DSTROOT=build/Archive",
                    "CFLAGS=$(CC_Flags) -isystembuild/Products/",
                    "LDFLAGS=$(LD_Flags) -Lbuild/Products/",
-                   "RC_ARCHS=x86_64", # Force 64-bit build for Mojave
+                   "RC_ARCHS=%s" % Hardware::CPU.arch,
```

Comment:
`brew style` prefers it like this:
```suggestion
                   "RC_ARCHS=#{Hardware::CPU.arch}",
```

Reply:
Got it, thanks 👍 Will bear that in mind for next time!